app/org/sagebionetworks/bridge/services/ActivityEventService.java

```diff
@@ -94,6 +121,23 @@ public void publishActivityEvent(ActivityEvent event) {
         return activityEventDao.getActivityEventMap(healthCode);
     }
 
+    public List<ActivityEvent> getActivityEventList(String healthCode) {
+        Map<String, DateTime> activityEvents = getActivityEventMap(healthCode);
+
+        List<ActivityEvent> activityEventList = Lists.newArrayList();
+        for (Map.Entry<String, DateTime> entry : activityEvents.entrySet()) {
+            DateTime timestamp = entry.getValue();
+            activityEventList.add(
+                    new ActivityEventImpl(
+                            null,
+                            entry.getKey(),
+                            null,
+                            timestamp == null ? null : timestamp.getMillis()
+                    ));
+        }
+        return activityEventList;
```

Comment:
We could just return a serialization of the DynamoActivityEvent, which would be less code. See e.g. Study.STUDY_LIST_WRITER which only returns two fields of the study.